src/main/java/hec/ensemble/TimeSeriesIdentifier.java

```diff
@@ -0,0 +1,17 @@
+package hec.ensemble;
+
+public class TimeSeriesIdentifier {
```

Comment:
I like this. If we have to do precipitation grids down the road, I think this will be a lifesaver.